modules/misc/news.nix

```
@@ -2252,6 +2252,15 @@ in
           '~/.taskrc'.
         '';
       }
+
+      {
+        time = "2021-11-29T15:15:59+00:00";
+        message = ''
```

Since this only applies to macOS:
```suggestion
        condition = hostPlatform.isDarwin;
        message = ''
```

---

Thanks, I applied the change.